backend/codesirius/internal_api/admin.py

```
@@ -0,0 +1,144 @@
+                    "</div>",
+                    raw_key,
+                    copy_button_html,
+                ),
+            )
+        else:
+            # For updates, use the normal save process
+            super().save_model(request, obj, form, change)
+
+            # Clear default messages and add our custom update message
+            storage = messages.get_messages(request)
+            for message in storage:
```

Ruff (0.8.2)

Loop control variable `message` not used within loop body

Rename unused `message` to `_message`

(B007)